### PR TITLE
fix(schema): Make username/email case-insensitive

### PR DESCRIPTION
**Pull request by alfrunes**
Ticket: MEN-3524
Changelog: Title

Signed-off-by: Alf-Rune Siqveland <alf.rune@northern.tech>

### CODE REVIEW

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline

File: jwt/mocks/Handler.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Northern.tech AS
```
Comment (tranchitella) [inline jwt/mocks/Handler.go:1]: No need to bump the copyright year here

**Review by tranchitella — APPROVED** [review]

File: main.go [hunk 2]
```diff
@@ -120,6 +122,15 @@ func doMain(args []string) {
 
 			Action: runMigrate,
 		},
+		{
+			Name:  "maintenance",
+			Usage: "Maintenance commands",
+			Subcommands: cli.Commands{{
+				Name:   "usernames2lower",
```
Comment (tranchitella) [inline main.go:129]: sorry, I have to tell you that this CLI subcommand looks ugly to me. :)

**Review by tranchitella — APPROVED** [review]

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline

**Review by tranchitella — APPROVED** [review]
It looks good, I see you decided to drop both the CLI command and the migration.
We'll have to make sure we instruct the users on how to fix their database if they have mixed case usernames, though. What about expanding the commit message about this topic?
The comment for `jwt/mocks/Handler.go` still applies.

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline

**Pull request merged**